maintainers/maintainer-list.nix

```
@@ -8637,6 +8770,7 @@
   piegames = {
     name = "piegames";
     email = "nix@piegames.de";
+    matrix = "@piegamesde:matrix.org";
```

```suggestion
    matrix = "@piegames:matrix.org";
```

I'd be interested in knowning how you got that handle. There are some `piegamesde` Matrix accounts out there, but not on matrix.org.